### PR TITLE
Update User Guide For Add Command

### PR DESCRIPTION
**Pull request by iamtr**
This pull request updates the User Guide accordingly based on the latest capabilities of the add command, as well as make some minor modifications to replace addressbook with Nursing Address Book

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -110,27 +112,27 @@ Tags: FallRisk, Diabetes
...
-### Listing all patients : `list`
+### Listing patients : `list`
+
+Displays a list of patients in NAB. 
 
-Displays a list of all registered patients.
+By using list with optional parameters, you will be able to list all patients.
 
-Format: `list`, `list [w\WARD] [t\TAG]...`
+Optionally, you can filter by ward or tags to list only patients in a specific ward or with specific tags.
```
Comment (ejnan) [inline docs/UserGuide.md]: Maybe we avoid using 'you' for user guide?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -110,27 +112,27 @@ Tags: FallRisk, Diabetes
...
 
-Example command:
 
 `list`
 ```
-Here are the details of the 2 patients in your contact book:
-
-1. John Doe
-2. Jane Doe
+Listed all persons
```
Comment (ejnan) [inline docs/UserGuide.md:132]: Perhaps the output for the example command of `list` should actually show the whole list?

**Review by ejnan — CHANGES_REQUESTED** [review]
LGTM, just a few nits regarding the format and output for list

File: docs/UserGuide.md (hunk 1 above)
Comment (erv-teo) [reply docs/UserGuide.md]: i think part of the rubrics is to use you-language to enhance engagement

File: docs/UserGuide.md (hunk 2 above)
Comment (iamtr) [reply docs/UserGuide.md:132]: I have amended, does the change look good?

Comment (erv-teo) [reply docs/UserGuide.md:132]: i think you missed out remarks and the index, and also Date of Birth is just DOB in our app. 
```
1. John Doe
IC: T1234567P
DOB: 21 Mar 2000
Ward: A1
Admission Date: 2 Feb 2022
Remarks: 
Tags: FallRisk, Diabetes
```
Here, I also did some reordering to match the order in our app

File: docs/UserGuide.md [hunk 3]
```diff
@@ -110,27 +112,27 @@ Tags: FallRisk, Diabetes
...
 ```
 
-Example command:
 
 `list w\B4 t\SeverAllergies`
```
Comment (ryanlimdx) [inline docs/UserGuide.md]: This should be "SevereAllergies".

File: docs/UserGuide.md [hunk 4]
```diff
@@ -144,11 +146,11 @@ Ward: B4
 Tags: SevereAllergies
```
Comment (ryanlimdx) [inline docs/UserGuide.md]: Similar to what Ervin said, it should be DOB and Remarks is missing

**Review by ejnan — APPROVED** [review]
LGTM!

**Pull request merged**